rbac/migration_tool/sharedSystemRolesReplicatedRoleBindings.py

```diff
@@ -97,13 +97,12 @@ def set_system_roles(cls):
 
 def v1_role_to_v2_mapping(
```

Comment:
It looks like there are some other callers still passing in more arguments we'd need to update the signature for: https://github.com/RedHatInsights/insights-rbac/pull/1180/files#diff-1dbf098bdd583306097f07e14314357a5949c2c93a1cfd1d562cbc426f6099d9R260

Reply:
Ah, nice catch. Will fix.

Reply:
Looks like that other callsite actually wasn't used (it just forwarded to another method). So I was able to just remove it.